test.sh

```diff
@@ -108,8 +161,6 @@ function initializeCert() {
      -keystore $JRE_HOME/lib/security/cacerts \
```

Comment:
it looks like `JRE_HOME` for this container is actually `/usr/lib/jvm/java-8-openjdk-amd64/jre/`